### PR TITLE
Added code to exit if file doesnt exist

### LINKED ISSUE #3: Exit gracefully if the file to be compiled doesn't exist
Provide proper error message and exit gracefully.
[refer](https://github.com/FOSS-Cell-GECPKD/interpreter/blob/master/src/lox.py#L20-L24)

### PR DESCRIPTION
**Pull request by AthulJoshi**
Is this good?
Fix #3

### CODE REVIEW

File: src/lox.py [hunk 1]
```diff
@@ -17,10 +17,14 @@ def run(self, source):
...
-        # TODO: If the file doesn't exist
+        # If the file doesn't exist
         # exit gracefully
-        with open(path) as f:
-            script = f.read()
+        try:
+            with open(path) as f:
+                script = f.read()
+        except IOError:
+            print("Sorry, the file doesn't exist!")
```
Comment (twentyse7en) [inline src/lox.py]: We don't need to say `Sorry` :sweat_smile: . It's better to specify the file name in the error message. Hope this helps :)

**Review by twentyse7en — CHANGES_REQUESTED** [review]

**Comment by AthulJoshi** [comment]
Okay will change that :D

File: src/lox.py (hunk 1 above)
Comment (AthulJoshi) [reply src/lox.py]: Okay fixing that.. :)

**Comment by AthulJoshi** [comment]
Is it okay now? 😁

File: src/lox.py [hunk 2]
```diff
@@ -17,10 +18,14 @@ def run(self, source):
...
-        # TODO: If the file doesn't exist
+        # If the file doesn't exist
         # exit gracefully
-        with open(path) as f:
-            script = f.read()
+        try:
+            with open(path) as f:
+                script = f.read()
+        except IOError:
+            print(f"Can't open {basename(path)} : No such file exists")
```
Comment (twentyse7en) [inline src/lox.py:27]: :+1:

**Review by twentyse7en — CHANGES_REQUESTED** [review]
It's a good practice to combine the commits if they below to the same purpose. As you only changed the message in the second commit it's better to combine those. [This](https://levelup.gitconnected.com/how-to-squash-git-commits-9a095c1bc1fc) might help you.

**Comment by twentyse7en** [comment]
Thx for contributing!

**Pull request merged**